react/week2/to_do_list/src/index.js

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+
+const todos = [
+  {
+    id: 0,
+    description: "Get out of bed",
+    color: "brown",
+    deadline: "2020-06-08",
+    done: false,
+    Visibility: true
```

Comment:
```suggestion
    visibility: true
```
Only components should be capitalised